tests/cross_functional/kcs/test_monitor_recovery.py

```diff
@@ -112,7 +112,7 @@ def mon_recovery_setup(
             data=self.object_data,
         ), "Failed: PutObject"
 
-    def test_monitor_recovery(
+    def deprecated_test_monitor_recovery(
```

Comment:
This is an active test and we should not be deprecating it

Reply:
done